src/main/scala/org/tensorframes/impl/datatypes.scala

```diff
@@ -199,7 +199,7 @@ private[tensorframes] sealed abstract class ScalarTypeOperation[@specialized(Int
   @deprecated("to remove", "now")
   def convertBuffer(buff: ByteBuffer, numElements: Int): Iterable[Any]
 
-  def convertTensor(t: tf.Tensor): MWrappedArray[T]
+  def convertTensor(t: tf.Tensor[_]): MWrappedArray[_]
```

Comment:
It would be nice to add type safety now that tensorflow adds the type of the underlying tensor.

Reply:
you mean you would want to restrict the type of Tensor?

Reply:
Yes, but let's do that in the future.